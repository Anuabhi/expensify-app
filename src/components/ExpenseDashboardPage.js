import React from 'react';
import ReactDOM from 'react-dom';
import ExpenseList from './ExpenseList';
import ExpenseListItem from './ExpenseListItem';
import ExpenseListFilters from './ExpenseListFilters';



const ExpenseDashboardPage =() =>(
    <div>
    This is from my Dashboard components
    

    <ExpenseListFilters />


    <ExpenseList />
    
    </div>

);

export default ExpenseDashboardPage;