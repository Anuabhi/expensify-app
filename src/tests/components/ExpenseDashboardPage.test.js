import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixture/expenses';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('should render ExpenseList with expenses',()=>{
    const wrapper=shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
})