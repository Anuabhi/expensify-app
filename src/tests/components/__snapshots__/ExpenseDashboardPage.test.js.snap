// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with expenses 1`] = `
<div>
  This is from my Dashboard components
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
